### PR TITLE
docs: Add AvailableFrom label

### PR DESCRIPTION
**Pull request by patrycjakalinska**
This PR introduces the `AvailableFrom` label (a standard version and a full-sized version). The purpose of this label is to inform users that feature is available starting from a specified version and in all subsequent versions.

https://github.com/user-attachments/assets/172c4930-75fd-41f1-8616-845e0832cdd1

### CODE REVIEW

File: packages/docs-reanimated/src/components/AvailableFrom/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  const getBadgeContent = () => {
+    if (isFull) {
+      return (
+        <>
+          <div className={styles.infoIcon}>
+            {isLightMode ? <Danger /> : <DangerDark />}
+          </div>
+          <span>
+            Available from Reanimated{' '}
+            <span className={styles.version}>v{version}</span>
```
Comment (tomekzaw) [inline packages/docs-reanimated/src/components/AvailableFrom/index.tsx:28]: Let's drop `v` prefix here for better readability
```suggestion
            <span className={styles.version}>{version}</span>
```

Comment (patrycjakalinska) [reply packages/docs-reanimated/src/components/AvailableFrom/index.tsx:28]: Should we drop `v` prefix for both full version and normal?

Comment (tomekzaw) [reply packages/docs-reanimated/src/components/AvailableFrom/index.tsx:28]: I think yes.

**Review by piaskowyk — APPROVED** [review]
Looks great for me 👍

**Pull request merged**